test/evm/hardhat/SpokePool.ExecuteRootBundle.ts

```diff
@@ -1,4 +1,5 @@
 import { SignerWithAddress, seedContract, toBN, expect, Contract, ethers, BigNumber } from "../../../utils/utils";
+import { buildRelayerRefundMerkleTree } from "../../svm/utils";
```

Comment:
this is great, I really like this pattern that you have here being able to mix the EVM/SVM tests via the tree building logic.